README.md

```diff
@@ -121,7 +122,9 @@ process at our hosting provider will unpack these symlinks to actual files
 during the deployment process.
 
 Please note, symlinks should only be created between integration domain
-directories. The `_placeholder` & `_homeassistant` directories are special
+folders (ie. don't create symlinks for files), and also only between integration domain folders within the same main folder (ie. don't symlink between the custom and core integrations folders).
```

Comment:
This is incorrect, we don't allow symlinks in any custom integration at this point.

Maybe we should change the first sentence instead:

`symlinking domain folders for the same icon/logos is allowed.`

to:

`symlinking domain folders for the same icon/logos is only allowed for core integrations.`

Reply:
Ok, I'll update the text, though note that [custom_integrations/indego](https://github.com/home-assistant/brands/blob/master/custom_integrations/indego) appears to be symlinked to bosch, which is why I thought it may have been ok to symlink.

Reply:
Sure, sometimes we learn about things that we did in the past.